CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Added
 
+- Fix organization glimpse card title for accessibility purposes
```

Comment:
Can you move this entry into the `Fix` section please.

Reply:
'course I can.